packages/governance/src/committeeRegistrar.js

```diff
@@ -26,7 +26,7 @@ const start = zcf => {
   };
 
   const makeCommitteeVoterInvitation = index => {
-    const handler = voterSeat => {
+    const handler = Far('handler', voterSeat => {
```

Comment:
Is this right? Should we now mark all of the offerHandlers with Far now?

Reply:
As of this PR, and for the immediate future, no. That's why I'm making a special case for makeInvitation. Longer term, maybe. It's a tradeoff. But I think yes.

Reply:
Ok, I'll create a documentation issue for explaining the usage, in case users see examples like this, but I'll say it isn't required